tests/02_eradiate/02_system/test_ckd_basic.py

```diff
@@ -16,6 +16,21 @@ def test_ckd_basic(modes_all_ckd):
     We check for correctness when using CKD modes with atmosphere-free scenes.
     This test is designed to check that the CKD infrastructure
     (preprocessing, spectral loop, postprocessing) works.
+
+    Rationale
+    ---------
+
+    * Sensor: Distant measure covering a plane (25 angular points,
+      10^6 sample per pixel) and targeting (0, 0, 0).
+    * Illumination: Directional illumination with a zenith angle
+      :math:`\theta = 0.0°` with black body irradiance spectrum.
```

Comment:
You'll have to escape this `\t`:
```suggestion
      :math:`\\theta = 0.0°` with black body irradiance spectrum.
```